server.js

```diff
@@ -0,0 +1,109 @@
+// EXPRESS ---------------------------------------------
+import express from 'express';
+const app = express();
+import './db-connect.js';
+import Student from './models/Student.js';
+
+    // MIDDLEWARE --------------------------------------
+    app.use( express.json() );
+
+
+    // ROUTES ------------------------------------------
+        // HOME
+        app.get("/", (req, res) => {
```

Comment:
Typically on top level we do not do indentation of code. We just use indentation when we actually "descend down", e.g. into an if block or into a callback.

To separate pure logical blocks, we typically just use spacing...

But who is "we"? Fuck the conventions!